components/Layout/Header/Header.jsx

```diff
@@ -1,25 +1,7 @@
-import HackneyLogo from './HackneyLogo';
+import { Header } from 'lbh-frontend-react';
```

Comment:
Is this loading the whole library in the final bundle?

Reply:
Yeah I double checked the .next build and it's not.